### ISSUE TITLE
I exported a json file from 【spine 2d】, but, I try to 【import local file】from aftereffect、bodymovin. But, it showed: import failed.  

### ISSUE THREAD

**Issue by jackjuniorarty** [airbnb/lottie-web#2708]
**Tell us about your environment**

* **Browser and Browser Version:**
* **After Effects Version:**

**What did you do? Please explain the steps you took before you encountered the problem.**

**What did you expect to happen?**

**What actually happened? Please include as much _relevant_ detail as possible.**

**Please provide a download link to the After Effects file that demonstrates the problem.**

**Comment by jackjuniorarty** [comment]
Hello, thank you for your reply ~
environment : 
Chrome : 97.0.4692.71
After effects: 2020
Spine : 4.0.57professional

(一) what did I do ? 
(1) I did a simple character animation in Spine ( a character with pictures、bones、animations ).
And I exported  it to a json file (with png).

(2) I opened a AE file with some special effects. 
I imported the json file in AE/ window / Extensions / Bodymovin / import Lottie Animation / Import Local file /
And it shows : Import Failed. Alerts (There has been an error).

What did I expect ?
I just expected that I can import json files with 2d character animations to Aftereffects. And the character animations which finished in spine application.
